test/test_cbe.py

```diff
@@ -168,6 +168,12 @@ def test_consistent_return_value_c(test_filename, tmpdir, cflags):
     print('regular executable returned', regular_retval)
     assert regular_retval in [TEST_SUCCESS_EXIT_CODE, TEST_XFAIL_EXIT_CODE]
 
+    # suppress "array subscript -1 is outside array bounds" in test expected
+    # to trigger it
+    if "test_char_sized_ptr_math_decr" in test_filename:
```

Comment:
I know this part isn't optimal, but right now we only need to do it for one test. Adding a whole mechanism for test-specific flags feels a bit overkill for now.

Reply:
This seems good. If there was a way to write arbitrary strings to the cbe_c file, then it could add:
```
#pragma GCC diagnostic ignored "-Warray-bounds"
```
But I don't think there is any particularly clean way to do that?